mods/d2k/rules/world.yaml

```diff
@@ -35,6 +35,14 @@
 		Name: Corrino
 		InternalName: corrino
 		Selectable: false
+	Faction@Mercenaries:
+		Name: Mercenaries
+		InternalName: mercenaries
+		Selectable: false
+	Faction@Smugglers:
+		Name: Smugglers
+		InternalName: smuggler
```

Comment:
In the original game the internal name for smugglers was `smuggler`, so I used that, but I couldn't find out the internal name for the mercs.

I now looked at the d2k exe a moment ago and the internal name for mercs is indeed singular `mercenary` as well.

Does it make a difference to the map importer? If yes, I'd change `mercenaries` to `mercenary`, if no, I'd change `smuggler` to `smugglers`.